components/treeview/stories/treeview.stories.js

```diff
@@ -104,8 +248,15 @@ Default.args = {
 		},
 	],
 };
+Disabled.parameters = {
+	chromatic: { disableAllSnapshots: true },
+};
 
-// ********* DOCS ONLY ********* //
+/**
+ * Icons can be used to add clarification about tree view items. These help to signify content types, which creates easier reference and context within the hierarchy.
```

Comment:
There's no Icons variant! It was so similar to this one that it seemed a little silly. Open to discussion though!